management_interface/management_interface/urls.py

```diff
@@ -17,5 +17,9 @@
 from django.urls import path
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
+    path("admin/", admin.site.urls),
 ]
+
+admin.site.site_header = "HANS Management Interface"
+admin.site.site_title = "HANS Management Interface"
+admin.site.index_title = "Welcome to HANS Management Interface"
```

Comment:
I'd like at least *one* of these to have the words "Hospital", "Activity", "Notification", and "Service" in it if they fit.  No idea which one is best.  Can we give this a try and see if it doesn't look too ridiculous?

```suggestion
admin.site.index_title = "Welcome to the Hospital Activity Notification Service"
```